scipy/stats/_multivariate.py

```diff
@@ -3943,6 +3943,7 @@ class multivariate_t_gen(multi_rv_generic):
 
     Examples
     --------
+    >>> import numpy as np
```

Comment:
last I knew when the docs build they have `numpy` already setup by default. If you're running the module in your machine as a stand alone file you would need to add that but as far as I recall we haven't changed so safe to remove this line.

Reply:
Even if we can, should we? Why remove lines that are needed to run example code?

Reply:
We can remove the import--the debate about copy-paste viability is perhaps another matter, but I'm pretty sure the convention in PR reviews has been to discourage import `numpy as np` over and over.

Reply:
Okay, I've removed the `numpy` import.